app/src/main/java/com/example/where2meet/SignupActivity.java

```diff
@@ -0,0 +1,84 @@
+package com.example.where2meet;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Toast;
+
+import com.example.where2meet.databinding.ActivitySignupBinding;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+import com.parse.SignUpCallback;
+
+public class SignupActivity extends AppCompatActivity {
+    private ActivitySignupBinding activitySignupBinding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        activitySignupBinding = ActivitySignupBinding.inflate(getLayoutInflater());
+        View view = activitySignupBinding.getRoot();
+        setContentView(view);
+
+
+        activitySignupBinding.tvLogin.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(SignupActivity.this, LoginActivity.class);
+                startActivity(intent);
```

Comment:
should you close the signup activity? what happens if you click back on login after doing this?